openbb_platform/providers/oecd/openbb_oecd/models/ppi.py

```diff
@@ -0,0 +1,112 @@
+"""OECD Producer Price Index Model."""
+
+import re
+from datetime import date, timedelta
+from typing import Any, Dict, List, Literal, Optional, Union
+
+from openbb_core.provider.abstract.fetcher import Fetcher
+from openbb_core.provider.standard_models.ppi import (
+    ProducerPriceIndexData,
+    ProducerPriceIndexQueryParams,
+)
+from openbb_oecd.utils import constants, helpers
+from pydantic import Field, field_validator
+
+countries = tuple(constants.COUNTRY_TO_CODE_PPI.keys())
+CountriesLiteral = Literal[countries]  # type: ignore
+
+
+class OECDProducerPriceIndexQueryParams(ProducerPriceIndexQueryParams):
+    """OECD Producer Price Index Query."""
+
+    country: CountriesLiteral = Field(
```

Comment:
Unit of measurement can be: [Annual growth rate (%), 2015=100]

There is a "perspective" parameter: [Total Market, Domestic Market]

Reply:
Intentionally left out for scope here.